testnets/elystestnet/chain.json

```diff
@@ -29,17 +27,36 @@
         "denom": "uelys",
         "fixed_min_gas_price": 0.01,
         "low_gas_price": 0.01,
-        "average_gas_price": 0.025,
+        "average_gas_price": 0.02,
         "high_gas_price": 0.03
+      },
+      {
+        "denom": "ibc/F082B65C88E4B6D5EF1DB243CDA1D331D002759E938A0F5CD3FFDC5D53B3E349",
+        "fixed_min_gas_price": 0.001,
+        "low_gas_price": 0.001,
+        "average_gas_price": 0.002,
+        "high_gas_price": 0.003
+      },
+      {
+        "denom": "ibc/C4CFF46FD6DE35CA4CF4CE031E643C8FDC9BA4B99AE598E9B0ED98FE3A2319F9",
```

Comment:
There's still one more asset that needs to be registered.
`Error: Chain elystestnet does not have fee token ibc/C4CFF46FD6DE35CA4CF4CE031E643C8FDC9BA4B99AE598E9B0ED98FE3A2319F9 defined in its Assetlist.`